catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/util/UiTestUtils.java

```diff
@@ -53,6 +54,13 @@ public static String getResourcesString(int stringId) {
 		return InstrumentationRegistry.getTargetContext().getResources().getString(stringId);
 	}
 
+	public static boolean isRTL(Locale locale) {
```

Comment:
if you are using this function only in the activity/rtl package, please make a new utility class there and move this function.

Reply:
done